### PR TITLE
[WIP] Ref conv kernel for Quantized conv 

### PR DESCRIPTION
**Pull request by nbpatel**
Not mature yet. Needs generalizing and some edge case handling.

### CODE REVIEW

File: src/ngraph/op/experimental/quantized_conv.cpp [hunk 1]
```diff
@@ -40,15 +40,17 @@ op::QuantizedConvolution::QuantizedConvolution(const shared_ptr<Node>& data_batc
     , m_data_dilation_strides(data_dilation_strides)
     , m_requantize(requantize)
 {
-    constructor_validate_and_infer_types();
-
     //TODO(nbpatel): Add checks.
```
Comment (diyessi) [inline src/ngraph/op/experimental/quantized_conv.cpp]: This should all be done in `validate_and_infer_types`, which is when element type and shape propagation will happen in the near future. Likewise, you need to be calling `constructor_validate_and_infer_types` which will become a no-op and then be removed in the near future, but, currently, does the type/shape propagation.

File: src/ngraph/runtime/reference/quantized_convolution.hpp [hunk 2]
```diff
@@ -0,0 +1,240 @@
+//*****************************************************************************
+// Copyright 2017-2019 Intel Corporation
```
Comment (diyessi) [inline src/ngraph/runtime/reference/quantized_convolution.hpp]: I think I might be able to get the other convolution template to do this.

**Comment by adstraw** [comment]
I created some more unit tests for QLinearConv in PR #2705 .  Let's plan to merge the changes here through that PR.

**Comment by nbpatel** [comment]
#2840

**Pull request closed without merging**